### PR TITLE
Sources watching refactoring

### PR DESCRIPTION
**Pull request by Cactucs**
In previous version I didn't liked rewriting the variables...

### CODE REVIEW

File: lib/tasks/watch_sources.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-var mapping = {
+var _ = require('lodash')
```
Comment (enzy) [inline lib/tasks/watch_sources.js]: we put all require calls into function body below to minimalize startup time

Comment (Cactucs) [inline lib/tasks/watch_sources.js]: fixed!

**Comment by enzy** [comment]
Please rebase this on master and review the changes.

**Comment by Cactucs** [comment]
IMO no conflict with https://github.com/manGoweb/mango-cli/commit/939805d11ff6eac1b556f5bfc22f81738940a7f1

**Pull request merged**